components/network/tuntap/tuntap-docs.p5m

```diff
@@ -0,0 +1,27 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2016 Jim Klimov
+#
+
+set name=pkg.fmri value=pkg:/driver/network/tuntap-docs@$(IPS_COMPONENT_VERSION),$(BUILD_VERSION)
```

Comment:
Please, rename it to driver/network/tuntap/documentation... It seems to be a more common pattern.